integ/s3/validate-and-clean.go

```diff
@@ -121,6 +131,9 @@ func validate(s3Client *s3.S3, response *s3.ListObjectsV2Output, bucket string,
 			if d == "" {
 				continue
 			}
+			if len(d) > 500 {
+				continue
```

Comment:
Maybe I am missing something. Why do we need this condition and the upper limit is 500?

Reply:
Look at the new S3 logger, it spits out log lines that are just numbers, and also large lines of 1KB random data (because S3 can only send data in chunks of a few MB). I wanted to make it so that the code didn't waste time trying to process those lines with random data.

500 is an arbitrary number between the size of each type of log event. The integer lines that we count will be smaller and the random data lines will be larger.